server/app/models/grid_domain_authorization.rb

```diff
@@ -18,6 +27,22 @@ class GridDomainAuthorization
 
   # @return [String]
   def to_path
-    "#{self.grid.try(:name)}/#{self.name}"
+    "#{self.grid.try(:name)}/#{self.domain}"
   end
+
+  def computed_state
```

Comment:
My suggestion would be to name this kind of "computed state" as the `status`. Same as for the `HostNode.status`